README.md

```diff
@@ -1,13 +1,20 @@
-# jkennedy-research-and-reflection
+# Jacob Kennedy - Research & Reflection
 
+## Course goals
+
+<!-- Context 1 -->
 Over the time in this course, I want to take actionable steps to become a stronger programmer and more in tune and efficient with what I do when developing. I often write code and create programs and find myself kicking myself when I see others do the same tasks so much more efficiently than I. I want to work towards having a deep understanding of the effects of what I'm doing when writing programs and how I interact with the machine I work on. 
 
-I'm going to spend time on coding websites and online communites challenging myself to write more complex and meaningful code on a daily basis. I want to spend at least 8 hours a week focusing specifically on programming in Java.
+<!-- Goal 1 -->
+I'm going to spend time on coding websites and online communites challenging myself to write more complex and meaningful code on a daily basis. I want to spend at least 8 hours a week focusing specifically on programming in Java. 
 
+<!-- Goal 2 -->
 In the theme of 104 being a app development course, I'd like to increase my knowledge of mobile platforms and learn some useful practices for efficient coding on mobile platforms. I'm going to research people and teams who have experience in the field of mobile development and study/learn from them using resources like youtube.
```

Comment:
This is also a great goal. The course will handle at least part of this second goal (useful practices and efficient coding on mobile platforms). 

There is some relationship between the two goals, so you could choose to combine them somewhat (perhaps via the measure as discussed above), or this second goal could be more in line with your community code project. For this one, I would encourage you to think a bit more about the specifics (maybe you could choose some small number of YouTube channels that provide mobile app dev resources and compare them).